app/src/main/java/com/kamron/pogoiv/MainActivity.java

```diff
@@ -2,6 +2,7 @@
 
 import android.Manifest;
 import android.annotation.TargetApi;
```

Comment:
@sarav You are completely right about this one.
I probably didn't split up this commit into 4 different one just because I hadn't the time to do that.
There's an easy way to split it now?